javax/src/test/java/org/jboss/ejb/client/DiscoveryEJBClientInterceptorTestCase.java

```diff
@@ -31,21 +31,21 @@
 public class DiscoveryEJBClientInterceptorTestCase {
 
     @Test
-    public void testBlackList() throws Exception {
+    public void testBlocklist() throws Exception {
         long timeout = 1000L;
-        System.setProperty("org.jboss.ejb.client.discovery.blacklist.timeout", timeout + "");
+        System.setProperty("org.jboss.ejb.client.discovery.blocklist.timeout", timeout + "");
```

Comment:
can you change the property name to: `SystemProperties.DISCOVERY_BLOCKLIST_TIMEOUT`  to avoid repeating its literal value?